chapter_preliminaries/autograd.md

```diff
@@ -1,50 +1,50 @@
 ```{.python .input}
 %load_ext d2lbook.tab
-tab.interact_select(['mxnet', 'pytorch', 'tensorflow'])
+tab.interact_select(['mxnet', 'pytorch', 'tensorflow', 'jax'])
 ```
 
 # Automatic Differentiation
+
 :label:`sec_autograd`
 
-Recall from :numref:`sec_calculus` 
+Recall from :numref:`sec_calculus`
 that calculating derivatives is the crucial step
 in all of the optimization algorithms
 that we will use to train deep networks.
 While the calculations are straightforward,
-working them out by hand can be tedious and error-prone, 
+working them out by hand can be tedious and error-prone,
 and this problem only grows
 as our models become more complex.
 
 Fortunately all modern deep learning frameworks
 take this work off of our plates
-by offering *automatic differentiation*
-(often shortened to *autograd*). 
+by offering _automatic differentiation_
+(often shortened to _autograd_).
 As we pass data through each successive function,
-the framework builds a *computational graph* 
+the framework builds a _computational graph_
 that tracks how each value depends on others.
-To calculate derivatives, 
-automatic differentiation packages 
+To calculate derivatives,
+automatic differentiation packages
 then work backwards through this graph
-applying the chain rule. 
+applying the chain rule.
 The computational algorithm for applying the chain rule
-this fashion is called *backpropagation*.
+this fashion is called _backpropagation_.
```

Comment:
Let's not change these.

Reply:
Those whitespaces are a nice cleanup but it will make an unnecessarily long diff when we finally merge the jax branch to master at some point in time, making everything harder to review.

I'd suggest not to touch the text unless very necessary or if you find a very obvious mistake etc.
Let's stick to code with comments as discussed!

Reply:
I assume these were changed by a formatter that I use on my local. I will revert them.